conformance/tests/conformance_rbac.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: v1
```

Comment:
for consistency with the manifests in /deploy, can we use `-` instead of `_` in the name?